testing/piknik/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=piknik
+pkgver=0.9.1
+pkgrel=0
+pkgdesc="Copy/paste anything over the network"
+url="https://github.com/jedisct1/piknik"
+arch="all"
+license="BSD-2-Clause"
+makedepends="go glide"
+source="$pkgname-$pkgver.tar.gz::https://github.com/jedisct1/piknik/archive/$pkgver.tar.gz"
+builddir="$srcdir/src/github.com/jedisct1/$pkgname"
+
+prepare() {
+        mkdir -p ${builddir%/*}  
+        mv "$srcdir"/$pkgname-$pkgver "$builddir"/
+        cd "$builddir"
+	export GOPATH="$srcdir"
+	glide install
```

Comment:
Looks like a combination of tabs and spaces.

Reply:
Fixed, thanks